src/Auth0Client.ts

```diff
@@ -114,6 +114,7 @@ export default class Auth0Client {
 
     // Don't use web workers unless using refresh tokens in memory and not IE11
     if (
+      window.Worker &&
```

Comment:
@adamjmcgrath What do you think? Seeing as we now have both paths available (worker and non-worker) I feel like adding a sanity check here makes sense.

Reply:
Well, it will do once the tests are fixed. Looking at them now.

Reply:
Fixed! Also added a test for when there is no worker available.